conductr_cli/resolvers/bintray_resolver.py

```diff
@@ -0,0 +1,153 @@
+BINTRAY_CREDENTIAL_FILE_PATH = '{}/.bintray/.credentials'.format(os.getenv('HOME'))
+
+
+def resolve_bundle(cache_dir, uri):
+    try:
+        bintray_username, bintray_password = load_bintray_credentials()
+        urn, org, repo, package_name, compatibility_version, digest = bundle_shorthand.parse(uri)
+        log('Resolving bundle', org, repo, package_name, compatibility_version, digest)
+        bundle_download_url = bintray_download_url(bintray_username, bintray_password, org, repo, package_name,
+                                                   compatibility_version, digest)
+        if bundle_download_url:
+            return uri_resolver.resolve_bundle(cache_dir, bundle_download_url)
+        else:
+            return False, None, None
+    except MalformedBundleUriError:
+        return False, None, None
+    except HTTPError:
+        return False, None, None
+
+
+def load_from_cache(cache_dir, uri):
+    try:
+        bintray_username, bintray_password = load_bintray_credentials()
+        urn, org, repo, package_name, compatibility_version, digest = bundle_shorthand.parse(uri)
+        log('Loading bundle from cache', org, repo, package_name, compatibility_version, digest)
+        bundle_download_url = bintray_download_url(bintray_username, bintray_password, org, repo, package_name,
+                                                   compatibility_version, digest)
+        if bundle_download_url:
+            return uri_resolver.load_from_cache(cache_dir, bundle_download_url)
+        else:
+            return False, None, None
+    except MalformedBundleUriError:
+        return False, None, None
+    except HTTPError:
+        return False, None, None
+
+
+def load_bintray_credentials():
+    if not os.path.exists(BINTRAY_CREDENTIAL_FILE_PATH):
+        print('Bintray credentials not found in {}'.format(BINTRAY_CREDENTIAL_FILE_PATH))
+        return None, None
+    else:
+        with open(BINTRAY_CREDENTIAL_FILE_PATH, 'r') as cred_file:
+            lines = [line.replace('\n', '') for line in cred_file.readlines()]
+            data = dict()
+            for line in lines:
+                key, value = line.replace(' = ', '=').split('=')
+                data[key] = value
+
+            username = None if 'user' not in data else data['user']
+            password = None if 'password' not in data else data['password']
+            print('Bintray credentials loaded from {}'.format(BINTRAY_CREDENTIAL_FILE_PATH))
+            return username, password
+
+
+def bintray_download_url(bintray_username, bintray_password, org, repo, package_name, compatibility_version, digest):
+    if compatibility_version is None and digest is None:
+        # Get latest version
+        package_endpoint = '{}/packages/{}/{}/{}'.format(BINTRAY_API_BASE_URL, org, repo, package_name)
+        package = get_json(package_endpoint, bintray_username, bintray_password)
+        latest_version = package['latest_version']
```

Comment:
@huntc - I've implemented the change to infer latest version from list of attribute names that starts with `latest-v` below